salt/metalk8s/orchestrate/control_plane.sls

```diff
@@ -0,0 +1,65 @@
+{%- set control_plane_ips = salt.saltutil.runner('mine.get', tgt='*', fun='control_plane_ips') %}
+{%- if pillar['bootstrap_id'] in control_plane_ips.keys() and control_plane_ips[pillar['bootstrap_id']] %}
+{%- set control_plane_ip = control_plane_ips[pillar['bootstrap_id']][0] %}
+{%- else %}
+{%- set control_plane_ip = 'localhost' %}
+{%- endif %}
+
+{% set node = pillar['node_name'] %}
+
+Update mine:
+  salt.state:
+    - tgt: {{ node }}
+    - saltenv: {{ saltenv }}
+    - sls:
+      - metalk8s.mine.ips
+
+Bootstrap client certs:
+  salt.state:
+    - tgt: {{ node }}
+    - saltenv: {{ saltenv }}
+    - sls:
+      - metalk8s.kubeadm.init.certs.apiserver
+      - metalk8s.kubeadm.init.certs.apiserver-etcd-client
+      - metalk8s.kubeadm.init.certs.apiserver-kubelet-client
+      - metalk8s.kubeadm.init.certs.front-proxy-client
+    - require:
+      - salt: Update mine
+    - pillar:
+        repo:
+          online_mode: false
```

Comment:
we don't need to override the 2 values: online_mode and local_mode

Reply:
Thx done.